src/components/course/Course.jsx

```diff
@@ -63,9 +66,13 @@ export default function Course() {
         userSubsidyApplicableToCourse,
         catalog,
         algoliaSearchParams,
+        courseRecommendations: {
+          allRecommendations: allRecommendations.slice(0, 3),
```

Comment:
We are assuming that all the recommendations will be in ascending order relevant to skills?

Reply:
@sameenfatima78 Can we add a `TODO` here to explain we will be filtering based on catalogs?

Reply:
@zamanafzal Yes, API that Hammad wrote already returns us the results in sorted order.
@HammadAhmadWaqas TODO is already added in the service file.